README.md

```diff
@@ -1,2 +1,145 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://www.tailwindcss.com/">Tailwind CSS</a></li>
+  </ul>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Signup**
+- **Login**
+- **Add Category**
+- **Add Transactions**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### 🚀 Live and Video Demo <a name="live and video-demo"></a>
+
+- [ video demo link](https://www.loom.com/share/0175cb699ec040a8bf745978105a47c5)
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- IDE or code editor: **VsCode, atom, sublime**...
+- To have [Ruby](https://www.ruby-lang.org/en/) and [Git](https://git-scm.com/) installed on your system.
+- To have [Node.js](https://nodejs.org/) installed on your system.
+
+After installing Ruby, run the following command to install rails.-
+
+```sh
+ gem install rails
+```
+
+### Setup
+
+Clone this project on your local machine using the following command in your terminal:
+
+```sh
+  git clone https://github.com/DaveZag/recipe-app.git
+```
+
+### Install
+
+Run the following command to install all project's dependencies:
+
+```sh
+  cd rails-app
+  bundle install
+  node install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Example command:
+
+```sh
+  rails server
+```
+
+OR
+
+```sh
+  rails s
+```
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [omarbabou](https://github.com/omarbabou)
+- LinkedIn: [Idrissa makoba omar](https://www.linkedin.com/in/idrissa-makoba-omar/)
+- twitter: [BabouOmar3](https://twitter.com/BabouOmar3)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add GUI**
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, kindly give it a star ⭐️
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Microverse
+- Ruby on Rails Documentation🙃
+- Every person who inspired this codebase.
+- Original design idea by [Gregoire Vella on Behance](https://www.behance.net/gregoirevella). Thanks also [The Creative Commons license](https://creativecommons.org/licenses/by-nc/4.0/).
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
[REQUIRED] I noticed that you deleted the License file instead of implementing the suggestion made by the previous reviewer. Kindly replace and correct the file